frontends/web/src/components/toggle/toggle.module.css

```diff
@@ -1,54 +1,50 @@
 .container {
-    position: relative;
-    display: inline-block;
-    width: 60px;
-    min-height: var(--item-height-xsmall);
-    margin: 0;
+  display: inline-block;
+  margin: 0;
+  min-height: var(--item-height-xsmall);
+  position: relative;
+  width: 60px;
 }
 
 .container input {
-    opacity: 0;
-    width: 0;
-    height: 0;
+  height: 0;
+  opacity: 0;
+  width: 0;
 }
 
 .container input[disabled] + .slider {
-    opacity: 0.4;
-    cursor: default;
+  cursor: default;
+  opacity: 0.4;
 }
 
 .slider {
-    position: absolute;
-    cursor: pointer;
-    top: 0;
-    left: 0;
-    right: 0;
-    bottom: 0;
-    background-color: var(--color-disabled);
-    -webkit-transition: .2s;
-    transition: .2s;
-    border-radius: 2px;
+  background-color: var(--color-disabled);
+  border-radius: 2px;
+  bottom: 0;
+  cursor: pointer;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  transition: .2s;
```

Comment:
removed prefixed CSS property `-webkit-transition` as we only target chromium browser and the build would automatically prefix if it was still necessary